README.md

```diff
@@ -9,7 +9,7 @@ The shared library manages all validation checks and constraints are defined in
 TODO - Analyze value of pip support
 
 ```
-pip install odevalidator
+pip install .
```

Comment:
Yeah changing it to this makes sense given that these are instructions for how to install the library itself. I'll close PR #7.